proto/def.bzl

```diff
@@ -83,14 +83,23 @@ def _go_proto_library_impl(ctx):
         compilers = ctx.attr.compilers
     go_srcs = []
     valid_archive = False
+
+    if ctx.attr.proto:
```

Comment:
I think it should be an error to specify both `proto` and `protos`: one or the other should be required. This should have been the case for `compilers` as well, but let's leave that aside for now.

Reply:
Done